### DISCUSSION TITLE
InfoWindow is loaded and not showing on the map

### DISCUSSION THREAD

**Discussion by vireakpanha-khy** [JustFly1984/react-google-maps-api#2115] [Show and tell]
Can anyone show me how to display the info window when clicking on the marker? I've tried the code from documentation and several YouTube videos but nothing works. The 'onLoad' function of InfoWindow is called but it is not showing on the map.

**Comment by JustFly1984** [comment]
please provide minimal reproduction in codesandbox.io

**Reply by vireakpanha-khy** [reply]
Sorry for the inconvenience. I've figured it out now. It's actually because of my import statement.

Here's what I wrote: `import { InfoWindow } from '../../node_modules/@react-google-maps/api/dist/index';`
And this is the solution:  `import { InfoWindow } from '@react-google-maps/api';`
